src/Components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { PaginationButton } from '../PaginationButton';
+
+export const Pagination = ({ total, itemsOnPage, page, onChangePage }) => {
+  const maxPages = Math.ceil(total / itemsOnPage);
+  const getValue = pageNum => ((
+    (pageNum === page - 2 && pageNum !== 1)
+        || (pageNum === page + 2 && pageNum !== maxPages)
+  )
+    ? '...'
+    : pageNum.toString());
+
+  return (
+    <nav className="p-2 m-auto">
+      <ul className="pagination">
+        <li
+          className={classNames('page-item', { disabled: page === 1 })}
+        >
+          <PaginationButton
+            simbol="&laquo;"
+            page={`${page - 1}`}
+            onPageChange={onChangePage}
+          />
+        </li>
+
+        {Array
+          .from({ length: maxPages }, (_, i) => i + 1)
+          .filter(pageNum => pageNum === 1
+                        || pageNum === maxPages
+                        || (pageNum >= page - 2 && pageNum <= page + 2))
+          .map(pageNum => (
```

Comment:
It is strongly recommended to avoid placing complex operations inside returned `jsx` element.
It's better to prepare it in external method and just apply map here.